api/src/main/java/org/openmrs/notification/impl/SentMessageServiceImpl.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.notification.impl;
+
+import java.util.List;
+
+import org.openmrs.User;
+import org.openmrs.api.APIException;
+import org.openmrs.api.impl.BaseOpenmrsService;
+import org.openmrs.notification.SentMessage;
+import org.openmrs.notification.SentMessageService;
+import org.openmrs.notification.db.SentMessageDAO;
+
+/**
+ * default implementation for SentMessageService
+ * @see org.openmrs.notification.SentMessageService
+ */
+public class SentMessageServiceImpl extends BaseOpenmrsService implements SentMessageService {
+
+	private SentMessageDAO dao;
+	
+	/**
+	 * default constructor
+	 */
+	public SentMessageServiceImpl() {}
```

Comment:
@rkorytkowski i thought the spring container uses these default constructors to instantiate beans. I saw a lot of them in previous code so i had to research why, and every link suggested that reason. Please correct me if am wrong